pyproject.toml

```diff
@@ -22,6 +21,8 @@ dependencies = [
 [project.scripts]
 meds_etl_mimic = "meds_etl.mimic:main"
 meds_etl_omop = "meds_etl.omop:main"
+meds_etl_flat = "meds_etl.flat:convert_flat_to_meds_main"
```

Comment:
these entrypoints need updating to match the new names (`meds_etl_from_flat` and `meds_etl_to_flat`)

Reply:
Oops, forgot to change these. Fixed now!